cli/Valet/Brew.php

```diff
@@ -15,12 +15,10 @@ class Brew
         'php@7.2',
         'php@7.1',
         'php@7.0',
-        'php@5.6',
         'php73',
         'php72',
         'php71',
         'php70',
-        'php56',
```

Comment:
This probably requires a new major version release?

Reply:
@driesvints Yah.. good call, thank you.